### DISCUSSION TITLE
这就是你所说的工程啊，@hollowaykeanho？

### DISCUSSION THREAD

**Discussion by MinXinS** [ZORALab/Hestia#175] [Q&A] [answered]
很大型啊。您肯定做得来吗？

**Comment by hollowaykeanho** [comment, marked as answer]
难也得做，已经被外人欺负够了。是时候开发咱们自己的科技了。

**Comment by MinXinS** [comment]
你这个https://github.com/ZORALab/Hestia/discussions/101 言论真的把群队搞到乱七八糟的。那么肯定要放弃Rust那么牛吗？

**Comment by hollowaykeanho** [comment]
意见如何？我又不是乱乱来。

**Comment by MinXinS** [comment]
觉得你应该被理会他们继续用Rust咯。Nim很稀见，怕没人会用。

再说呢，微型控制器还需要看厂家啊。如果他们一律支持Rust, 你也没办法啊。

**Comment by hollowaykeanho** [comment]
问题在Rust的大队实在太离谱了。我又不是不想用，而是他们确实往AMP的后步走啊。我真的不想在好像Bissetii一样重写Hestia啊。

> 再说呢，微型控制器还需要看厂家啊。如果他们一律支持Rust, 你也没办法啊。

支持了才说。

**Comment by MinXinS** [comment]
> 支持了才说。

够牛。

> 问题在Rust的大队实在太离谱了。我又不是不想用，而是他们确实往AMP的后步走啊。我真的不想在好像Bissetii一样重写Hestia啊。

暂时留着，别删除它。留条后路总好过。要不研究如何把Rust的RAII给拔出来放入Nim里用吧。

**Comment by hollowaykeanho** [comment]
> 暂时留着，别删除它。留条后路总好过。

这交叉点比赌博还要痛苦。

> 要不研究如何把Rust的RAII给拔出来放入Nim里用吧。

一旦放开1.2.0就开工了。

**Comment by MinXinS** [comment]
> 这交叉点比赌博还要痛苦。

少来这套，你好命。就试试看吧。这又不是最牛的事。

**Comment by MinXinS** [comment]
好的，一路顺风。离开了啊。
